core/src/main/java/org/dst/core/DstAbstractMap.java

```diff
@@ -0,0 +1,13 @@
+package org.dst.core;
+
+public interface DstAbstractMap<K, V> {
+
+  void put(K key, V value);
+
+  void remove(K key);
```

Comment:
How ablout return the value of delete

Reply:
Do we need the return value ? I have checked the code in Dst, there is no need to keep the return value of `remove()` method.

Reply:
I think it's better to give it a return value as well.

Reply:
Done.